cmd/libs/go2idl/informer-gen/generators/informer.go

```diff
@@ -134,7 +134,7 @@ type $.type|private$Informer struct {
 `
 
 var typeInformerConstructorInternal = `
-func new$.type|public$Informer(client $.clientSetInterface|raw$, resyncPeriod $.timeDuration|raw$) $.cacheSharedIndexInformer|raw$ {
+func new$.type|public$Informer(client $.clientSetInterface|raw$, resyncCheck, resyncPeriod $.timeDuration|raw$) $.cacheSharedIndexInformer|raw$ {
```

Comment:
I'm fairly certain you can spitball an algorithm that does a reasonable job of not letting resync periods go too long and hide that complexity.   Is there an example of a controller that really cares about its resync check period (in a way that isn't exactly the same as all other controllers)?